arbitrum-docs/partials/_troubleshooting-bridging-partial.md

```diff
@@ -0,0 +1,46 @@
+### How do I move assets between One and Nova? {#how-do-i-move-assets-between-one-and-nova}
+<p>Both Arbitrum One and Arbitrum Nova run as layers on top of Ethereum. Thus, you can always move assets between the two chains in two steps by going "through" Ethereum. In other words: withdraw your assets on Arbitrum One to Ethereum and then deposit them onto Nova, or conversely, withdraw your assets from Nova on to Ethreum and then deposit them on to Arbitrum One. These steps can all be done at <a href="https://bridge.arbitrum.io/">https://bridge.arbitrum.io/</a>.</p>
+
+<p>Some <a href="https://portal.arbitrum.one/#bridges">third party bridges</a> support transferring between Arbitrum One and Nova directly; (remember that third party bridges are created by third parties; DYOR!)</p>
```

Comment:
correct me if I'm wrong but as far as I know, there is no fast bridge that supports direct transfers between one and nova

Reply:
[Multichain](https://app.multichain.org/) supports transfers between One and Nova, for ETH and a few more tokens, but I'm afraid is the only one that supports that atm.
Do you think we should remove that paragraph? ETH transfers are interesting though